.github/workflows/trivy_scan.yml

```diff
@@ -0,0 +1,163 @@
+name: Scan Theia Docker images for vulnerabilities every day
+
+on:
+  schedule:
+    # every day at 10am
+    - cron: '0 10 * * *'
+  workflow_dispatch:
+
+jobs:
+  build:
+    if: github.repository == 'antrea-io/theia'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Find greatest Antrea version
+      id: find-theia-greatest-version
+      env:
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        VERSION=${{ github.event.inputs.theia-version }}
```

Comment:
you are missing the `theia-version` input in the `workflow_dispatch` definition above